tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/wizard/NewLiferayPluginProjectWizard.java

```diff
@@ -155,66 +157,73 @@ protected void performPostFinish()
         super.performPostFinish();
 
         final NewLiferayPluginProjectOp op = element().nearest( NewLiferayPluginProjectOp.class );
-        final IProject project = CoreUtil.getProject( op.getFinalProjectName().content() );
```

Comment:
Oops, you have deleted a call to this method but it isn't called in your new code?  Did you make sure to account for this potential loss of functionality?